EIPS/eip-draft_permission_token.md

```diff
@@ -0,0 +1,317 @@
+---
+eip: draft_permission_token
+title: Permission Token
+description: A new token that held the permission of an address in an ecosystem
+author: Chiro (@chiro-hiro)
+discussions-to: https://ethereum-magicians.org/t/eip-644-a-standard-for-permission-token/9105
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-19
+---
+
+## Abstract
+
+A new token standard that held the permission of an address in an ecosystem. A permission token can be transferred/delegated by the permission owner to an grantee's address.
+
+## Motivation
+
+You may still encounter special roles like `Owner`, `Operator`, `Manager`, `Validator` in many existing smart contracts because we're still using permissioned addresses to operate and manage them. It is difficult to audit and maintain since these permissions are not managed centrally in a single smart contract.
```

Comment:
No 2nd person:
```suggestion
Special roles like `Owner`, `Operator`, `Manager`, `Validator` are common for many smart contracts because permissioned addresses are used to operate and manage them. It is difficult to audit and maintain these system since these permissions are not managed centrally in a single smart contract.
```
That second sentence is wrong: they *are* stored "centrally" in a single smart contract. Ignoring that minor falsehood, the rest of this paragraph is okay.